tests/containers/rootless_podman.pm

```diff
@@ -57,8 +55,8 @@ sub run {
         switch_cgroup_version($self, 2);
         select_serial_terminal;
 
-        validate_script_output 'podman info', sub { /cgroupVersion: v2/ };
-        validate_script_output "id $testapi::username", sub { /systemd-journal/ };
+        validate_script_output('podman info', sub { /cgroupVersion: v2/ }, fail_message => "podman not operating in cgroup v2");
+        validate_script_output("id $testapi::username", sub { /systemd-journal/ }, fail_message => "testuser must be part of systemd-journal group") if (is_leap("<16") || is_sle("<16"));
```

Comment:
Looking at line 40, I find this check actually rather useless and think we should remove it.

Reply:
We can remove both lines.  We do the check for cgroup's already and the check for systemd-journal is useless.

Reply:
The `podman info` I don't see being checked elsewhere. Can you point me to the place where this is done?

Reply:
We run `podman info` right after installing it.

Reply:
But we don't check the cgroupVersion in there, at least not that I can see it.

Reply:
We have a couple of tests, one for v1 and another for v2

Reply:
If you refer to `docker_cgroupv2_tests` and `podman_cgroupv2_tests`, those are the tests that enable cgroupv2.

The line I'm referring to is `validate_script_output('podman info', sub { /cgroupVersion: v2/ }, fail_message => "podman not operating in cgroup v2");` and I think this still needs to be there, because this literally checks if podman sees cgroupsv2.

Reply:
I interpret the approval as this conversation is resolved. Thank you!